tests/test_patch_wsi_dataset_deprecated.py

```diff
@@ -0,0 +1,256 @@
+# Copyright (c) MONAI Consortium
```

Comment:
I think this file should be a "rename" instead of a totally new file?
Maybe you can try with `git mv` ?

Thanks.

Reply:
@Nic-Ma I have already used `git mv` everywhere but I don't know why it is considering new file for this file but not for the other!
Anyways, even if someone use the regular `mv`, once it is staged, git automatically detects that it's a rename! I don't know why it's not happening here.

Reply:
Hi @ericspod @wyli ,

Do you know any possible reason for this problem?

Thanks in advance.